src/js/plugin/sdk/pipeline/PipelineModel.js

```diff
@@ -0,0 +1,48 @@
+
+class PipelineModel {
+
+  constructor() {
+    this.pipelines = new Map();
+  };
+
+  clearPipeline() {
+    this.pipelines.clear();
+  };
+
+  applyFilter(operationName, filterParameters) {
```

Comment:
We should also document that a pipeline can't handle more than one input parameter (could be an object or list though) as the output of one operation will be the input of the next operation.

Reply:
Where is the best place to document this? As a comment in this method?

Reply:
I'd add a JSDoc comment for now.